core/kmod/sn_netdev.c

```diff
@@ -494,7 +499,7 @@ static int sn_poll(struct napi_struct *napi, int budget)
 		/* last check for race condition.
 		 * see sn_enable_interrupt() */
 		if (rx_queue->dev->ops->pending_rx(rx_queue)) {
-			napi_reschedule(napi);
+			napi_schedule(napi); // Changed from napi_reschedule to napi_schedule
```

Comment:
```suggestion
			napi_schedule(napi);
```

Not sure the comment is really helpful.